uiowa/uiowa
# Admissions - Community Colleges

Resolves https://github.com/uiowa/uiowa/issues/4976

## To Test

- `ddev composer install && ddev blt frontend && ddev blt sync --site admissions.uiowa.edu --define drush.aliases.remote=admissions.test`
- There is a new community colleges taxonomy that webmasters should have access to add/edit/delete terms within. The term is public to visitors.
- On that taxonomy should be a description field (intro), contact information (long text), locations (multi text), "unsure major" (long text with default value).
- On render the contact information field label should prepend the term name.
- List of majors will be handled in a follow-up.
- Webmasters should have the ability to place the community colleges block in layout builder which displays the colleges and the locations in the list.
- Looks like Percy is struggling with lazy loaded images again...

<img width="458" alt="Screen Shot 2022-04-01 at 1 37 04 PM" src="https://user-images.githubusercontent.com/4663676/161569021-2e910f4b-5070-4b1f-899a-9a42bc59d0da.png">
<img width="598" alt="Screen Shot 2022-04-04 at 8 58 23 AM" src="https://user-images.githubusercontent.com/4663676/161569029-85411a07-2911-4be0-a611-d067d091e2ec.png">
